sqrl-examples/avro/README.md

```diff
@@ -0,0 +1,68 @@
+# Avro for SQRL: Setup and Execution Guide
+
+## Compile SQRL Script
+Compile your .sqrl and schema.graphqls using Docker:
+```shell
+docker run --rm -v $PWD:/build datasqrl/cmd compile c360.sqrl schema.graphqls --mnt $PWD
+```
+
+## Add Kafka Topics
+Create a script to add orders and ordercount-1 topics in Kafka:
+```shell
+echo '
+#!/bin/bash
+/opt/bitnami/kafka/bin/kafka-topics.sh --create --bootstrap-server kafka:9092 --topic orders --partitions 1 --replication-factor 1
+/opt/bitnami/kafka/bin/kafka-topics.sh --create --bootstrap-server kafka:9092 --topic ordercount-1 --partitions 1 --replication-factor 1
+exit 0;' > build/deploy/create-topics.sh
+```
+## Modify Docker Compose
+Update Docker Compose for Kafka with external connection capabilities:
+
+```yaml
```

Comment:
Should we just add that to the docker-compose we generate to make it easier for users to test with Kafka?

Reply:
yeah sounds correct, ill add it